tests/backend/test_apiv1_organization_status.py

```diff
@@ -73,6 +74,21 @@ async def test_organization_update_expiration_date(
     assert rep == {"status": "ok"}
     rep = await organization_status(administration_backend_sock, coolorg.organization_id)
     assert rep == {"status": "ok", "is_bootstrapped": True, "expiration_date": None}
+    # Expired organization
+    with backend.event_bus.listen() as spy:
+        rep = await organization_update(
+            administration_backend_sock,
+            coolorg.organization_id,
+            expiration_date=Pendulum(1999, 12, 31),
+        )
+        assert rep == {"status": "ok"}
+        rep = await organization_status(administration_backend_sock, coolorg.organization_id)
+        assert rep == {
+            "status": "ok",
+            "is_bootstrapped": True,
+            "expiration_date": Pendulum(1999, 12, 31),
+        }
+        await spy.wait_with_timeout(BackendEvent.ORGANIZATION_EXPIRED)
```

Comment:
the wait_with_timeout should be moved before `organization_status` given it comes from the use of `organization_update`

Reply:
good catch